bin/cli.js

```diff
@@ -147,6 +154,35 @@ Example
       runTransform(target, command, program, { component, prop })
     );
 
+  program
+    .command(
+      "replace-prop-value <target> <component> <prop> <oldValue> <newValue>"
+    )
+    .description(
+      `
+Codemod used for prop value changing
+Usage
+  npx carbon-codemod replace-prop-value <target> <component> <prop> <oldValue> <newValue>
```

Comment:
We should also cater for named imports, e.g. `import {TileFooter} from ...`

Reply:
It already would wouldn't it? I'm not really sure what you mean?

Reply:
It's just looking for import path, it doesn't matter if it's named or not

Reply:
Did you create a ticket to do this?

Reply:
I did indeed, FE-3229